user/serializers.py

```diff
@@ -2,0 +1,29 @@
+from django.contrib.auth import get_user_model
+from rest_framework import serializers
+
+
+class UserSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = get_user_model()
+        fields = ("id", "username", "email", "password", "is_staff",)
+        read_only_fields = ("id", "is_staff",)
+        extra_kwargs = {
+            "password": {
+                "write_only": True,
+                "min_length": 5,
+            },
+        }
+
+    def create(self, validated_data):
+        """create user with encrypted password"""
+        return get_user_model().objects.create_user(**validated_data)
+
+    def update(self, instance, validated_data):
+        """update user with encrypted password"""
```

Comment:
The docstring for the `update` method should be capitalized and punctuated correctly. Consider changing it to: 'Update a user with an encrypted password.'